spec/support/api_spec_helper.rb

```diff
@@ -78,6 +78,13 @@ def interpolate_string value, options = {}
       named_record(m[1]).api_id
     elsif m = value.match(/^@valueOf\((.*), (.*)\)$/)
       named_record(m[1]).send(m[2])
+    elsif m = value.match(/^@date\((.*)\)$/)
+      if m[1].match(/(today|now)/)
+        Time.now.strftime('%Y-%m-%d')
+      else
+        m = value.match(/(\d+) (second|minute|hour|day|week|month|year)s? ago/)
```

Comment:
This line should be combined with the `else` into:

```
elsif m = value.match(/(\d+) (second|minute|hour|day|week|month|year)s? ago/)
```

And you should add an `else` that does something like:

```
else
  raise "Unknown date expectation format"
end
```